CHANGELOG.rst

```diff
@@ -6,7 +6,10 @@ This document describes changes between each past release.
 2.1 (unreleased)
 ----------------
 
-- Nothing changed yet.
+Fixed
+=====
+
+- Include all .j2 files in the packaged version (#308)
```

Comment:
IMHO, the changelog audience is users/admins, not devs.
it should contain entries on what actually change for users/admin.

Thus, I'd suggest a wording like:

> Fix "IOERrorr" crash running `ihatemoney generate-config`

(sorry if it seems like nitpicking)

Reply:
That's an interesting point. I'll update the branch, thanks for the comments, they're greatly appreciated, don't worry :)